src/commands/ca_install.rs

```diff
@@ -0,0 +1,102 @@
+use std::fs::copy;
+use std::fs::File;
+use std::io::Read;
+use std::io::Write;
+use std::path::PathBuf;
+use std::process::Command;
+use std::process::Stdio;
+
+use clap::App;
+use clap::Arg;
+use clap::ArgMatches;
+use clap::SubCommand;
+use dirs;
+use runas::Command as SudoCommand;
+
+use crate::config::certificates;
+
+pub(crate) fn command_config<'a, 'b>() -> App<'a, 'b> {
+    SubCommand::with_name("server:ca:install")
+        .name("server:ca:install")
```

Comment:
We can make it even easier 😉 
```suggestion
        .name("server:ca:install")
        .alias("ca:install")
```